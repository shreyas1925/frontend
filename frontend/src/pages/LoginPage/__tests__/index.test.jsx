import { LoginPage } from "../..";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
describe("LoginPage", () => {
    it("should render correctly", () => {
        const { asFragment } = render(<BrowserRouter><LoginPage /></BrowserRouter>);
        expect(asFragment()).toMatchSnapshot();
    })
    it("should render and have text", () => {
        const { getByText } = render(<BrowserRouter><LoginPage /></BrowserRouter>);
        expect(getByText("Login")).toBeInTheDocument();
    })
})