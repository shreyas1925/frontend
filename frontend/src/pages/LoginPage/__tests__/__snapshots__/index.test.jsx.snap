// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage should render correctly 1`] = `
<DocumentFragment>
  <div
    class="main--container"
  >
    <div
      class="left--container"
    >
      <div
        class="text--container"
      >
        <h1>
          Design APIs fast,
        </h1>
        <h1>
          Manage content easily
        </h1>
      </div>
      <div
        class="circle"
      />
      <div
        class="circle"
      />
      <div
        class="image--container"
      >
        <img
          alt=""
          src="people.png"
        />
      </div>
    </div>
    <div
      class="right--container"
    >
      <h1>
        Login to your CMS+ account
      </h1>
      <form
        action=""
        class="form--container"
      >
        <p>
          Email
        </p>
        <input
          autocomplete="off"
          name="email"
          type="email"
          value=""
        />
        <p>
          Password
        </p>
        <input
          autocomplete="off"
          name="password"
          type="password"
          value=""
        />
        <button
          type="submit"
        >
          Login
        </button>
        <p>
          Forgot password ? 
        </p>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
